wiki/core-rules.md

```diff
@@ -124,3 +124,45 @@ name : **nullary-unit**
 source : [NullaryUnit](/rules/core/src/main/scala/com/typesafe/abide/core/NullaryUnit.scala)
 
 It is not recommended to define methods with side-effects which take no arguments, as it is easy to accidentally invoke those side-effects.
+
+## Inferences of `Any` or `AnyVal`
```

Comment:
This doc seems misplaced, at least I couldn't find the implementation of this rule in this PR.. :-)